introduction.md

```diff
@@ -0,0 +1,144 @@
+---
```

Comment:
> Actually, again re the diagram i find it confusing it's in term of transactions and not outputs. Do you find it better conveys the flow that the PDF's diagram (which i find to be perfectly and immediately clear)?

I think they are both clear. Grouping into transactions and using descriptors abstraction matches the text better. Also this one shows which transactions are pre-signed (will make that and clock clearer with white background). What is confusing about it?

Reply:
> What is confusing about it?

What *i find* confusing is that it's in terms of transactions instead of outputs. What if a transaction pays to 3 deposit outputs? Also, is it clear the Cancel, UnEmer, and Spend all spend the same output?

Again, i'm not saying it's a valid reason to change it. Just expressing purely subjective feedback.

Reply:
Yeah good point. I think details of multiple outputs or batching are low-level and best left for transactions.md not the intro. But I can see how it could be unclear that the cancel, unemer and spend consume the same output. I'll extend the caption and make sure it's clear.